Docker/FreshRSS.Apache.conf

```diff
@@ -10,6 +10,11 @@ AllowEncodedSlashes On
 ServerTokens OS
 TraceEnable Off
 
+# Workaround to be able to check whether an environment variable is set
+# See: https://serverfault.com/questions/1022233/using-ifdefine-with-environment-variables/1022234#1022234
+Define VStart "${"
+Define VEnd "}"
+
```

Comment:
Can that be put inside the `<IfDefine OIDC_ENABLED>` block?

Reply:
I have not fully understood your response :-)

If possible, I would suggest the following:

```apache
<IfDefine OIDC_ENABLED>
	<IfModule !auth_openidc_module>
		Error "The auth_openidc_module is not available. Install it or unset environment variable OIDC_ENABLED."
	</IfModule>

	# Workaround to be able to check whether an OS environment variable is set
	# https://serverfault.com/questions/1022233/using-ifdefine-with-environment-variables/1022234#1022234
	Define VStart "${"
	Define VEnd "}"

	OIDCProviderMetadataURL ${OIDC_PROVIDER_METADATA_URL}
	OIDCClientID ${OIDC_CLIENT_ID}
	OIDCClientSecret ${OIDC_CLIENT_SECRET}

	OIDCRedirectURI /i/oidc/
	OIDCCryptoPassphrase ${OIDC_CLIENT_CRYPTO_KEY}

	Define "Test_${OIDC_REMOTE_USER_CLAIM}"
	<IfDefine Test_${VStart}OIDC_REMOTE_USER_CLAIM${VEnd}>
		OIDCRemoteUserClaim preferred_username
	</IfDefine>
	<IfDefine !Test_${VStart}OIDC_REMOTE_USER_CLAIM${VEnd}>
		OIDCRemoteUserClaim "${OIDC_REMOTE_USER_CLAIM}"
	</IfDefine>
	Define "Test_${OIDC_SCOPES}"
	<IfDefine Test_${VStart}OIDC_SCOPES${VEnd}>
		OIDCScope openid
	</IfDefine>
	<IfDefine !Test_${VStart}OIDC_SCOPES${VEnd}>
		OIDCScope "${OIDC_SCOPES}"
	</IfDefine>

	OIDCRefreshAccessTokenBeforeExpiry 30
</IfDefine>
```

Reply:
> I have not fully understood your response :-)

Apologies, I don't know how best to express this without actually showing what I mean in a diff. But I was referring to this: https://github.com/FreshRSS/FreshRSS/blob/666e951fa3e30c3a1b7f5ae68a2be4e06577b75d/Docker/Dockerfile#L68

The `OIDC_ENABLED`  variable is defined on the commandline inside each of the `Dockerfile`s, and it doesn't need to be if we use the same pattern for checking whether an environment variable is set or not.

I applied your suggestion anyway. Let me know if there's anything else :+1:

Reply:
This `OIDC_ENABLED` variable is special in the sense that, as far as I can see, it is the only one that changes the Apache command line (we want to add a `-D OIDC_ENABLED`), so I do not believe it is the same situation than the other environment variables